worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala

```diff
@@ -321,7 +320,7 @@ object DeviceMonitor {
       val freeSpace = usage(usage.length - 3)
       val used_percent = usage(usage.length - 2)
 
-      val status = freeSpace.toLong < RssConf.diskMinimumReserveSize(rssConf) / 1024 / 1024 / 1024
+      val status = freeSpace.toLong < rssConf.diskMinimumReserveSize / 1024 / 1024 / 1024
```

Comment:
I think we should rename rssConf to conf

Reply:
OK. I'll change it once and for all. 😎

Reply:
> OK. I'll change it once and for all. 😎

Also rename RssConf to CelebornConf?

Reply:
Sounds great. I'll change it.